src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -1,22 +1,593 @@
+    }
+  }, [errors, onConfigChange, onNext, formData])
 
   return (
     <div>
-      {title}
+      <Header
+        title="Garden Metadata"
+        subtitle="Fill with your garden information"
+      />
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          margin-bottom: ${6 * GU}px;
+        `}
+      >
+        <Field
+          label="Garden Name"
+          css={`
+            width: 100%;
+            marging-bottom: ${2 * GU}px;
+          `}
+        >
+          <TextInput
+            maxLength="15"
+            onChange={handleGardenNameChange}
+            value={formData.name}
+            required
+            css="width: 100%;"
+          />
+        </Field>
+        <Field
+          label="Garden Description"
+          css={`
+            width: 100%;
+            marging-bottom: ${2 * GU}px;
+          `}
+        >
+          <TextInput
+            multiline
+            maxLength="120"
+            onChange={handleGardenDescriptionChange}
+            value={formData.description}
+            required
+            css="width: 100%;"
+          />
+        </Field>
+        <Field
+          label="Assets"
+          css={`
+            width: 100%;
+          `}
+        >
+          <Box
+            css={`
+              width: 100%;
+            `}
+          >
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+                justify-content: space-around;
+              `}
+            >
+              <Field label="Garden Logo Type">
+                <Box
+                  css={`
+                    text-align: center;
+                    max-height: 143px;
+                  `}
+                >
+                  <ImageUploader
+                    id={1}
+                    onImageLoaded={handleOnGardenLogoTypeLoaded}
+                    onImageRemoved={handleOnGardenLogoTypeRemoved}
+                  />
+                </Box>
+              </Field>
+              <Field label="Garden Logo">
+                <Box
+                  css={`
+                    text-align: center;
+                  `}
+                >
+                  <ImageUploader
+                    id={2}
+                    onImageLoaded={handleOnGardenLogoLoaded}
+                    onImageRemoved={handleOnGardenLogoRemoved}
+                  />
+                </Box>
+              </Field>
+              <Field label="Token Icon">
+                <Box
+                  css={`
+                    text-align: center;
+                  `}
+                >
+                  <ImageUploader
+                    id={3}
+                    onImageLoaded={handleOnTokenLogoLoaded}
+                    onImageRemoved={handleOnTokenLogoRemoved}
+                  />
+                </Box>
+              </Field>
+            </div>
+          </Box>
+        </Field>
```

Comment:
```suggestion
		<Box
          heading="Assets"
          css={`
            width: 100%;
          `}
        >
          <div
            css={`
              display: flex;
              align-items: center;
              justify-content: space-around;
            `}
          >
            <Field label="Garden Logo Type">
              <Box
                css={`
                  text-align: center;
                  max-height: 143px;
                `}
              >
                <ImageUploader
                  id={1}
                  onImageLoaded={handleOnGardenLogoTypeLoaded}
                  onImageRemoved={handleOnGardenLogoTypeRemoved}
                />
              </Box>
            </Field>
            <Field label="Garden Logo">
              <Box
                css={`
                  text-align: center;
                `}
              >
                <ImageUploader
                  id={2}
                  onImageLoaded={handleOnGardenLogoLoaded}
                  onImageRemoved={handleOnGardenLogoRemoved}
                />
              </Box>
            </Field>
            <Field label="Token Icon">
              <Box
                css={`
                  text-align: center;
                `}
              >
                <ImageUploader
                  id={3}
                  onImageLoaded={handleOnTokenLogoLoaded}
                  onImageRemoved={handleOnTokenLogoRemoved}
                />
              </Box>
            </Field>
          </div>
        </Box>
 ```

Reply:
I test and removing the `Field` element prevents the issue that shows the upload modal when clicking any place of the `Field`. Note we will need some padding fixes afterwards.

Reply:
While we are at this. What you think of reducing the default size of the `ImageUploader` component now that we are just showing a green check?

Reply:
> I tested and removing the `Field` element prevents the issue that shows the upload modal when clicking any place of the `Field`. Note we will need some padding fixes afterwards.

hey gabi could you double check? just tested and still have the issue, about the size which size do you recommend?

Reply:
> I tested and removing the `Field` element prevents the issue that shows the upload modal when clicking any place of the `Field`. Note we will need some padding fixes afterwards.

hey gabi, could you double check? just tested what you suggested and still have the issue, about the size, which size do you recommend?